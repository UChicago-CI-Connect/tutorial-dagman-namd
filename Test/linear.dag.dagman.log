000 (063.000.000) 12/01 16:33:56 Job submitted from host: <128.135.112.71:30090?sock=1018_1873_3>
...
001 (063.000.000) 12/01 16:33:56 Job executing on host: <128.135.112.71:30090?sock=1018_1873_3>
...
005 (063.000.000) 12/01 16:39:45 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	0  -  Run Bytes Sent By Job
	0  -  Run Bytes Received By Job
	0  -  Total Bytes Sent By Job
	0  -  Total Bytes Received By Job
...
