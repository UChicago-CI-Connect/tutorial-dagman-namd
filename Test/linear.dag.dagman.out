12/01/14 16:33:56 Can't open directory "/home/condor/config" as PRIV_UNKNOWN, errno: 13 (Permission denied)
12/01/14 16:33:56 Cannot open /home/condor/config: Permission denied
12/01/14 16:33:56 ******************************************************
12/01/14 16:33:56 ** condor_scheduniv_exec.63.0 (CONDOR_DAGMAN) STARTING UP
12/01/14 16:33:56 ** /software/connect-connect-el6-x86_64/connect-client/bin/condor_dagman
12/01/14 16:33:56 ** SubsystemInfo: name=DAGMAN type=DAGMAN(10) class=DAEMON(1)
12/01/14 16:33:56 ** Configuration: subsystem:DAGMAN local:<NONE> class:DAEMON
12/01/14 16:33:56 ** $CondorVersion: 8.2.3 Sep 30 2014 BuildID: 274619 $
12/01/14 16:33:56 ** $CondorPlatform: x86_64_RedHat6 $
12/01/14 16:33:56 ** PID = 6362
12/01/14 16:33:56 ** Log last touched time unavailable (No such file or directory)
12/01/14 16:33:56 ******************************************************
12/01/14 16:33:56 Using config source: /home/balamurugan/.bosco/condor_config
12/01/14 16:33:56 Using local config sources: 
12/01/14 16:33:56    /home/balamurugan/.bosco/condor_config.local
12/01/14 16:33:56    /home/balamurugan/.bosco/config/condor_config.bosco_routing
12/01/14 16:33:56    /home/balamurugan/.bosco/config/condor_config.factory
12/01/14 16:33:56 config Macros = 99, Sorted = 98, StringBytes = 4216, TablesBytes = 3628
12/01/14 16:33:56 CLASSAD_CACHING is ENABLED
12/01/14 16:33:56 Daemon Log is logging: D_ALWAYS D_ERROR
12/01/14 16:33:56 SharedPortEndpoint: waiting for connections to named socket 6362_5abc
12/01/14 16:33:56 DaemonCore: command socket at <128.135.112.71:30090?sock=6362_5abc>
12/01/14 16:33:56 DaemonCore: private command socket at <128.135.112.71:30090?sock=6362_5abc>
12/01/14 16:33:56 DAGMAN_USE_STRICT setting: 1
12/01/14 16:33:56 DAGMAN_VERBOSITY setting: 3
12/01/14 16:33:56 DAGMAN_DEBUG_CACHE_SIZE setting: 5242880
12/01/14 16:33:56 DAGMAN_DEBUG_CACHE_ENABLE setting: False
12/01/14 16:33:56 DAGMAN_SUBMIT_DELAY setting: 0
12/01/14 16:33:56 DAGMAN_MAX_SUBMIT_ATTEMPTS setting: 6
12/01/14 16:33:56 DAGMAN_STARTUP_CYCLE_DETECT setting: False
12/01/14 16:33:56 DAGMAN_MAX_SUBMITS_PER_INTERVAL setting: 5
12/01/14 16:33:56 DAGMAN_USER_LOG_SCAN_INTERVAL setting: 5
12/01/14 16:33:56 DAGMAN_DEFAULT_PRIORITY setting: 0
12/01/14 16:33:56 DAGMAN_ALWAYS_USE_NODE_LOG setting: True
12/01/14 16:33:56 DAGMAN_SUPPRESS_NOTIFICATION setting: True
12/01/14 16:33:56 allow_events (DAGMAN_IGNORE_DUPLICATE_JOB_EXECUTION, DAGMAN_ALLOW_EVENTS) setting: 114
12/01/14 16:33:56 DAGMAN_RETRY_SUBMIT_FIRST setting: True
12/01/14 16:33:56 DAGMAN_RETRY_NODE_FIRST setting: False
12/01/14 16:33:56 DAGMAN_MAX_JOBS_IDLE setting: 0
12/01/14 16:33:56 DAGMAN_MAX_JOBS_SUBMITTED setting: 0
12/01/14 16:33:56 DAGMAN_MAX_PRE_SCRIPTS setting: 0
12/01/14 16:33:56 DAGMAN_MAX_POST_SCRIPTS setting: 0
12/01/14 16:33:56 DAGMAN_ALLOW_LOG_ERROR setting: False
12/01/14 16:33:56 DAGMAN_MUNGE_NODE_NAMES setting: True
12/01/14 16:33:56 DAGMAN_PROHIBIT_MULTI_JOBS setting: False
12/01/14 16:33:56 DAGMAN_SUBMIT_DEPTH_FIRST setting: False
12/01/14 16:33:56 DAGMAN_ALWAYS_RUN_POST setting: True
12/01/14 16:33:56 DAGMAN_ABORT_DUPLICATES setting: True
12/01/14 16:33:56 DAGMAN_ABORT_ON_SCARY_SUBMIT setting: True
12/01/14 16:33:56 DAGMAN_PENDING_REPORT_INTERVAL setting: 600
12/01/14 16:33:56 DAGMAN_AUTO_RESCUE setting: True
12/01/14 16:33:56 DAGMAN_MAX_RESCUE_NUM setting: 100
12/01/14 16:33:56 DAGMAN_WRITE_PARTIAL_RESCUE setting: True
12/01/14 16:33:56 DAGMAN_DEFAULT_NODE_LOG setting: @(DAG_DIR)/@(DAG_FILE).nodes.log
12/01/14 16:33:56 DAGMAN_GENERATE_SUBDAG_SUBMITS setting: True
12/01/14 16:33:56 DAGMAN_MAX_JOB_HOLDS setting: 100
12/01/14 16:33:56 DAGMAN_HOLD_CLAIM_TIME setting: 20
12/01/14 16:33:56 ALL_DEBUG setting: 
12/01/14 16:33:56 DAGMAN_DEBUG setting: 
12/01/14 16:33:56 argv[0] == "condor_scheduniv_exec.63.0"
12/01/14 16:33:56 argv[1] == "-Lockfile"
12/01/14 16:33:56 argv[2] == "linear.dag.lock"
12/01/14 16:33:56 argv[3] == "-AutoRescue"
12/01/14 16:33:56 argv[4] == "1"
12/01/14 16:33:56 argv[5] == "-DoRescueFrom"
12/01/14 16:33:56 argv[6] == "0"
12/01/14 16:33:56 argv[7] == "-Dag"
12/01/14 16:33:56 argv[8] == "linear.dag"
12/01/14 16:33:56 argv[9] == "-Suppress_notification"
12/01/14 16:33:56 argv[10] == "-CsdVersion"
12/01/14 16:33:56 argv[11] == "$CondorVersion: 8.2.3 Sep 30 2014 BuildID: 274619 $"
12/01/14 16:33:56 argv[12] == "-Dagman"
12/01/14 16:33:56 argv[13] == "/software/connect-connect-el6-x86_64/connect-client/bin/condor_dagman"
12/01/14 16:33:56 Default node log file is: </home/balamurugan/Work/Tutorial_Nov18th14/tutorial-dagman-namd/Test/./linear.dag.nodes.log>
12/01/14 16:33:56 DAG Lockfile will be written to linear.dag.lock
12/01/14 16:33:56 DAG Input file is linear.dag
12/01/14 16:33:56 Ignoring value of DAGMAN_LOG_ON_NFS_IS_ERROR.
12/01/14 16:33:56 Parsing 1 dagfiles
12/01/14 16:33:56 Parsing linear.dag ...
12/01/14 16:33:56 Dag contains 4 total jobs
12/01/14 16:33:56 Sleeping for 12 seconds to ensure ProcessId uniqueness
12/01/14 16:34:08 Bootstrapping...
12/01/14 16:34:08 Number of pre-completed nodes: 0
12/01/14 16:34:08 Of 4 nodes total:
12/01/14 16:34:08  Done     Pre   Queued    Post   Ready   Un-Ready   Failed
12/01/14 16:34:08   ===     ===      ===     ===     ===        ===      ===
12/01/14 16:34:08     0       0        0       0       1          3        0
12/01/14 16:34:08 0 job proc(s) currently held
12/01/14 16:34:08 Registering condor_event_timer...
12/01/14 16:34:09 MultiLogFiles: truncating log file /home/balamurugan/Work/Tutorial_Nov18th14/tutorial-dagman-namd/Test/./linear.dag.nodes.log
12/01/14 16:34:09 Submitting Condor Node A0 job(s)...
12/01/14 16:34:09 Adding a DAGMan auxiliary log /home/balamurugan/Work/Tutorial_Nov18th14/tutorial-dagman-namd/Test/./linear.dag.nodes.log
12/01/14 16:34:09 Masking the events recorded in the DAGMAN auxiliary log
12/01/14 16:34:09 Mask for auxiliary log is 0,1,2,4,5,7,9,10,11,12,13,16,17,24,27
12/01/14 16:34:09 submitting: condor_submit -a dag_node_name' '=' 'A0 -a +DAGManJobId' '=' '63 -a DAGManJobId' '=' '63 -a submit_event_notes' '=' 'DAG' 'Node:' 'A0 -a dagman_log' '=' '/home/balamurugan/Work/Tutorial_Nov18th14/tutorial-dagman-namd/Test/./linear.dag.nodes.log -a +DAGManNodesMask' '=' '"0,1,2,4,5,7,9,10,11,12,13,16,17,24,27" -a DAG_STATUS' '=' '0 -a FAILED_COUNT' '=' '0 -a +DAGParentNodeNames' '=' '"" -a +KeepClaimIdle' '=' '20 -a notification' '=' 'never namd_run_job0.submit
12/01/14 16:34:09 From submit: Submitting job(s).
12/01/14 16:34:09 From submit: 1 job(s) submitted to cluster 64.
12/01/14 16:34:10 	assigned Condor ID (64.0.0)
12/01/14 16:34:10 Just submitted 1 job this cycle...
12/01/14 16:34:10 Currently monitoring 1 Condor log file(s)
12/01/14 16:34:10 Reassigning the id of job A0 from (64.0.0) to (64.0.0)
12/01/14 16:34:10 Event: ULOG_SUBMIT for Condor Node A0 (64.0.0)
12/01/14 16:34:10 Number of idle job procs: 1
12/01/14 16:34:10 Of 4 nodes total:
12/01/14 16:34:10  Done     Pre   Queued    Post   Ready   Un-Ready   Failed
12/01/14 16:34:10   ===     ===      ===     ===     ===        ===      ===
12/01/14 16:34:10     0       0        1       0       0          3        0
12/01/14 16:34:10 0 job proc(s) currently held
12/01/14 16:34:25 Currently monitoring 1 Condor log file(s)
12/01/14 16:34:25 Event: ULOG_GRID_SUBMIT for Condor Node A0 (64.0.0)
12/01/14 16:35:30 Currently monitoring 1 Condor log file(s)
12/01/14 16:35:30 Event: ULOG_JOB_TERMINATED for Condor Node A0 (64.0.0)
12/01/14 16:35:30 Number of idle job procs: 0
12/01/14 16:35:30 Node A0 job proc (64.0.0) completed successfully.
12/01/14 16:35:30 Node A0 job completed
12/01/14 16:35:30 Of 4 nodes total:
12/01/14 16:35:30  Done     Pre   Queued    Post   Ready   Un-Ready   Failed
12/01/14 16:35:30   ===     ===      ===     ===     ===        ===      ===
12/01/14 16:35:30     1       0        0       0       1          2        0
12/01/14 16:35:30 0 job proc(s) currently held
12/01/14 16:35:35 Submitting Condor Node A1 job(s)...
12/01/14 16:35:35 Adding a DAGMan auxiliary log /home/balamurugan/Work/Tutorial_Nov18th14/tutorial-dagman-namd/Test/./linear.dag.nodes.log
12/01/14 16:35:35 Masking the events recorded in the DAGMAN auxiliary log
12/01/14 16:35:35 Mask for auxiliary log is 0,1,2,4,5,7,9,10,11,12,13,16,17,24,27
12/01/14 16:35:35 submitting: condor_submit -a dag_node_name' '=' 'A1 -a +DAGManJobId' '=' '63 -a DAGManJobId' '=' '63 -a submit_event_notes' '=' 'DAG' 'Node:' 'A1 -a dagman_log' '=' '/home/balamurugan/Work/Tutorial_Nov18th14/tutorial-dagman-namd/Test/./linear.dag.nodes.log -a +DAGManNodesMask' '=' '"0,1,2,4,5,7,9,10,11,12,13,16,17,24,27" -a DAG_STATUS' '=' '0 -a FAILED_COUNT' '=' '0 -a +DAGParentNodeNames' '=' '"A0" -a +KeepClaimIdle' '=' '20 -a notification' '=' 'never namd_run_job1.submit
12/01/14 16:35:35 From submit: Submitting job(s).
12/01/14 16:35:35 From submit: 1 job(s) submitted to cluster 65.
12/01/14 16:35:35 	assigned Condor ID (65.0.0)
12/01/14 16:35:35 Just submitted 1 job this cycle...
12/01/14 16:35:35 Currently monitoring 1 Condor log file(s)
12/01/14 16:35:35 Reassigning the id of job A1 from (65.0.0) to (65.0.0)
12/01/14 16:35:35 Event: ULOG_SUBMIT for Condor Node A1 (65.0.0)
12/01/14 16:35:35 Number of idle job procs: 1
12/01/14 16:35:35 Of 4 nodes total:
12/01/14 16:35:35  Done     Pre   Queued    Post   Ready   Un-Ready   Failed
12/01/14 16:35:35   ===     ===      ===     ===     ===        ===      ===
12/01/14 16:35:35     1       0        1       0       0          2        0
12/01/14 16:35:35 0 job proc(s) currently held
12/01/14 16:35:45 Currently monitoring 1 Condor log file(s)
12/01/14 16:35:45 Event: ULOG_GRID_SUBMIT for Condor Node A1 (65.0.0)
12/01/14 16:36:55 Currently monitoring 1 Condor log file(s)
12/01/14 16:36:55 Event: ULOG_JOB_TERMINATED for Condor Node A1 (65.0.0)
12/01/14 16:36:55 Number of idle job procs: 0
12/01/14 16:36:55 Node A1 job proc (65.0.0) completed successfully.
12/01/14 16:36:55 Node A1 job completed
12/01/14 16:36:55 Of 4 nodes total:
12/01/14 16:36:55  Done     Pre   Queued    Post   Ready   Un-Ready   Failed
12/01/14 16:36:55   ===     ===      ===     ===     ===        ===      ===
12/01/14 16:36:55     2       0        0       0       1          1        0
12/01/14 16:36:55 0 job proc(s) currently held
12/01/14 16:37:00 Submitting Condor Node A2 job(s)...
12/01/14 16:37:00 Adding a DAGMan auxiliary log /home/balamurugan/Work/Tutorial_Nov18th14/tutorial-dagman-namd/Test/./linear.dag.nodes.log
12/01/14 16:37:00 Masking the events recorded in the DAGMAN auxiliary log
12/01/14 16:37:00 Mask for auxiliary log is 0,1,2,4,5,7,9,10,11,12,13,16,17,24,27
12/01/14 16:37:00 submitting: condor_submit -a dag_node_name' '=' 'A2 -a +DAGManJobId' '=' '63 -a DAGManJobId' '=' '63 -a submit_event_notes' '=' 'DAG' 'Node:' 'A2 -a dagman_log' '=' '/home/balamurugan/Work/Tutorial_Nov18th14/tutorial-dagman-namd/Test/./linear.dag.nodes.log -a +DAGManNodesMask' '=' '"0,1,2,4,5,7,9,10,11,12,13,16,17,24,27" -a DAG_STATUS' '=' '0 -a FAILED_COUNT' '=' '0 -a +DAGParentNodeNames' '=' '"A1" -a +KeepClaimIdle' '=' '20 -a notification' '=' 'never namd_run_job2.submit
12/01/14 16:37:00 From submit: Submitting job(s).
12/01/14 16:37:00 From submit: 1 job(s) submitted to cluster 66.
12/01/14 16:37:00 	assigned Condor ID (66.0.0)
12/01/14 16:37:00 Just submitted 1 job this cycle...
12/01/14 16:37:00 Currently monitoring 1 Condor log file(s)
12/01/14 16:37:00 Reassigning the id of job A2 from (66.0.0) to (66.0.0)
12/01/14 16:37:00 Event: ULOG_SUBMIT for Condor Node A2 (66.0.0)
12/01/14 16:37:00 Number of idle job procs: 1
12/01/14 16:37:00 Of 4 nodes total:
12/01/14 16:37:00  Done     Pre   Queued    Post   Ready   Un-Ready   Failed
12/01/14 16:37:00   ===     ===      ===     ===     ===        ===      ===
12/01/14 16:37:00     2       0        1       0       0          1        0
12/01/14 16:37:00 0 job proc(s) currently held
12/01/14 16:37:10 Currently monitoring 1 Condor log file(s)
12/01/14 16:37:10 Event: ULOG_GRID_SUBMIT for Condor Node A2 (66.0.0)
12/01/14 16:38:20 Currently monitoring 1 Condor log file(s)
12/01/14 16:38:20 Event: ULOG_JOB_TERMINATED for Condor Node A2 (66.0.0)
12/01/14 16:38:20 Number of idle job procs: 0
12/01/14 16:38:20 Node A2 job proc (66.0.0) completed successfully.
12/01/14 16:38:20 Node A2 job completed
12/01/14 16:38:20 Of 4 nodes total:
12/01/14 16:38:20  Done     Pre   Queued    Post   Ready   Un-Ready   Failed
12/01/14 16:38:20   ===     ===      ===     ===     ===        ===      ===
12/01/14 16:38:20     3       0        0       0       1          0        0
12/01/14 16:38:20 0 job proc(s) currently held
12/01/14 16:38:25 Submitting Condor Node A3 job(s)...
12/01/14 16:38:25 Adding a DAGMan auxiliary log /home/balamurugan/Work/Tutorial_Nov18th14/tutorial-dagman-namd/Test/./linear.dag.nodes.log
12/01/14 16:38:25 Masking the events recorded in the DAGMAN auxiliary log
12/01/14 16:38:25 Mask for auxiliary log is 0,1,2,4,5,7,9,10,11,12,13,16,17,24,27
12/01/14 16:38:25 submitting: condor_submit -a dag_node_name' '=' 'A3 -a +DAGManJobId' '=' '63 -a DAGManJobId' '=' '63 -a submit_event_notes' '=' 'DAG' 'Node:' 'A3 -a dagman_log' '=' '/home/balamurugan/Work/Tutorial_Nov18th14/tutorial-dagman-namd/Test/./linear.dag.nodes.log -a +DAGManNodesMask' '=' '"0,1,2,4,5,7,9,10,11,12,13,16,17,24,27" -a DAG_STATUS' '=' '0 -a FAILED_COUNT' '=' '0 -a +DAGParentNodeNames' '=' '"A2" -a notification' '=' 'never namd_run_job3.submit
12/01/14 16:38:25 From submit: Submitting job(s).
12/01/14 16:38:25 From submit: 1 job(s) submitted to cluster 67.
12/01/14 16:38:25 	assigned Condor ID (67.0.0)
12/01/14 16:38:25 Just submitted 1 job this cycle...
12/01/14 16:38:25 Currently monitoring 1 Condor log file(s)
12/01/14 16:38:25 Reassigning the id of job A3 from (67.0.0) to (67.0.0)
12/01/14 16:38:25 Event: ULOG_SUBMIT for Condor Node A3 (67.0.0)
12/01/14 16:38:25 Number of idle job procs: 1
12/01/14 16:38:25 Of 4 nodes total:
12/01/14 16:38:25  Done     Pre   Queued    Post   Ready   Un-Ready   Failed
12/01/14 16:38:25   ===     ===      ===     ===     ===        ===      ===
12/01/14 16:38:25     3       0        1       0       0          0        0
12/01/14 16:38:25 0 job proc(s) currently held
12/01/14 16:38:35 Currently monitoring 1 Condor log file(s)
12/01/14 16:38:35 Event: ULOG_GRID_SUBMIT for Condor Node A3 (67.0.0)
12/01/14 16:39:45 Currently monitoring 1 Condor log file(s)
12/01/14 16:39:45 Event: ULOG_JOB_TERMINATED for Condor Node A3 (67.0.0)
12/01/14 16:39:45 Number of idle job procs: 0
12/01/14 16:39:45 Node A3 job proc (67.0.0) completed successfully.
12/01/14 16:39:45 Node A3 job completed
12/01/14 16:39:45 Of 4 nodes total:
12/01/14 16:39:45  Done     Pre   Queued    Post   Ready   Un-Ready   Failed
12/01/14 16:39:45   ===     ===      ===     ===     ===        ===      ===
12/01/14 16:39:45     4       0        0       0       0          0        0
12/01/14 16:39:45 0 job proc(s) currently held
12/01/14 16:39:45 All jobs Completed!
12/01/14 16:39:45 Note: 0 total job deferrals because of -MaxJobs limit (0)
12/01/14 16:39:45 Note: 0 total job deferrals because of -MaxIdle limit (0)
12/01/14 16:39:45 Note: 0 total job deferrals because of node category throttles
12/01/14 16:39:45 Note: 0 total PRE script deferrals because of -MaxPre limit (0)
12/01/14 16:39:45 Note: 0 total POST script deferrals because of -MaxPost limit (0)
12/01/14 16:39:45 Of 4 nodes total:
12/01/14 16:39:45  Done     Pre   Queued    Post   Ready   Un-Ready   Failed
12/01/14 16:39:45   ===     ===      ===     ===     ===        ===      ===
12/01/14 16:39:45     4       0        0       0       0          0        0
12/01/14 16:39:45 0 job proc(s) currently held
12/01/14 16:39:45 Wrote metrics file linear.dag.metrics.
12/01/14 16:39:45 Metrics not sent because of PEGASUS_METRICS or CONDOR_DEVELOPERS setting.
12/01/14 16:39:45 **** condor_scheduniv_exec.63.0 (condor_DAGMAN) pid 6362 EXITING WITH STATUS 0
